src/dune_pkg/archive_driver.mli

```diff
@@ -0,0 +1,17 @@
+open Stdune
+
+(** Knows how to extract archives of a particular format *)
+type t
+
+(** Driver for tarballs, possibly compressed *)
+val tar : t
+
+(** Driver for zip files *)
+val zip : t
+
+(** Return the driver that can extract a file of a given name. The decision is
+    made based on the file's suffix. *)
```

Comment:
```suggestion
(** Returns the driver that can extract a file of a given name. The decision is
    made based on the file's suffix. *)
```
Really nitpicky here, but for consistency as you used the `(* Knows [...]` above.